conversion/conversion.go

```diff
@@ -67,25 +67,63 @@ import (
 	dydb "github.com/aws/aws-sdk-go/service/dynamodb"
 	"github.com/aws/aws-sdk-go/service/dynamodbstreams"
 	mysqldriver "github.com/go-sql-driver/mysql"
+	"github.com/jackc/pgx/v5"
+	"github.com/jackc/pgx/v5/stdlib"
 	"go.uber.org/zap"
 	adminpb "google.golang.org/genproto/googleapis/spanner/admin/database/v1"
 	"google.golang.org/grpc/metadata"
 	"google.golang.org/protobuf/proto"
-	"github.com/jackc/pgx/v5"
-    "github.com/jackc/pgx/v5/stdlib"
 )
 
 var (
 	// Set the maximum number of concurrent workers during foreign key creation.
 	// This number should not be too high so as to not hit the AdminQuota limit.
 	// AdminQuota limits are mentioned here: https://cloud.google.com/spanner/quotas#administrative_limits
 	// If facing a quota limit error, consider reducing this value.
-	MaxWorkers       = 50
-	once             sync.Once
-	datastreamClient *datastream.Client
+	MaxWorkers          = 50
+	once                sync.Once
+	datastreamClient    *datastream.Client
+	resourcesForCleanup []*ConnectionProfileReq
```

Comment:
Why do we need this global variable?

Reply:
Because parallel tasks are used which cannot take any extra params and this seemed like the best way to add resources that were actually created.